pkg/operator/sync_v310_00.go

```diff
@@ -23,6 +23,100 @@ import (
 
 // most of the time the sync method will be good for a large span of minor versions
 func sync_v310_00_to_00(c ServiceCertSignerOperator, operatorConfig *scsv1alpha1.ServiceCertSignerOperatorConfig, previousAvailability *operatorsv1alpha1.VersionAvailablity) (operatorsv1alpha1.VersionAvailablity, []error) {
+	signingVersionAvailability, signingErrors := sync_v310_00_to_00_SigningController(c, operatorConfig, previousAvailability)
+	apiServiceInjectorVersionAvailability, apiServiceInjectorErrors := sync_v310_00_to_00_APIServiceController(c, operatorConfig, previousAvailability)
+
+	allErrors := []error{}
+	allErrors = append(allErrors, signingErrors...)
+	allErrors = append(allErrors, apiServiceInjectorErrors...)
+
+	mergedVersionAvailability := operatorsv1alpha1.VersionAvailablity{
+		Version: operatorConfig.Spec.Version,
+	}
+	mergedVersionAvailability.Generations = append(mergedVersionAvailability.Generations, signingVersionAvailability.Generations...)
+	mergedVersionAvailability.Generations = append(mergedVersionAvailability.Generations, apiServiceInjectorVersionAvailability.Generations...)
+	if signingVersionAvailability.UpdatedReplicas > 0 && apiServiceInjectorVersionAvailability.UpdatedReplicas > 0 {
+		mergedVersionAvailability.UpdatedReplicas = 1
+	}
+	if signingVersionAvailability.ReadyReplicas > 0 && apiServiceInjectorVersionAvailability.ReadyReplicas > 0 {
+		mergedVersionAvailability.ReadyReplicas = 1
+	}
+
+	return mergedVersionAvailability, allErrors
+}
+
+func sync_v310_00_to_00_APIServiceController(c ServiceCertSignerOperator, operatorConfig *scsv1alpha1.ServiceCertSignerOperatorConfig, previousAvailability *operatorsv1alpha1.VersionAvailablity) (operatorsv1alpha1.VersionAvailablity, []error) {
+	versionAvailability := operatorsv1alpha1.VersionAvailablity{
+		Version: operatorConfig.Spec.Version,
+	}
+
+	errors := []error{}
+	var err error
```

Comment:
drop this and just _, _,, err := ... below ?

Reply:
> drop this and just _, _,, err := ... below ?

as above, uniformity